en/guides/publishing-open-source-code.md

```diff
@@ -85,15 +85,15 @@ Recommended permissive licences are:
 |                                          | MIT | Apache 2.0 |
 | ---------------------------------------- | --- | ---------- |
 | **When to use** | Small, simple projects and scripts. | Larger software project use Apache 2.0 because it provides a grant of patents. |
-| **How to apply to your code** |  |  |
+| **How to apply to your code** |  <td colspan=2>Add the text of your chosen lisense to a LICENSE.txt file in the root folder of your project.  |
```

Comment:
You can't mix the `|` and HTML table syntax in GitHub. You'll need to either convert the entire table to HTML, or keep the empty cell